interval/query.go

```diff
@@ -57,6 +58,8 @@ func ReadToChan(inputFile string, send chan<- Interval) {
 		for val := range receive {
 			send <- val
 		}
+	default:
+		log.Fatalf("ERROR: file type of %s not supported by interval ReadToChan", inputFile)
```

Comment:
Not a huge deal but would it be possible to do: 
```suggestion
		log.Fatalf("Error: file type of %s not supported by interval ReadToChan", inputFile)
```
Capital letter + lowercase. Either way works for me, as long as we stay consistent with repo styling.